tests/ami/packages_sanity_tests.py

```diff
@@ -256,8 +256,14 @@ def _get_source_type(version_string):
         return "url"
     elif os.path.exists(version_string) and os.path.isfile(version_string):
         return "file"
-    else:
+    elif re.match(r"\d+\.\d+\.\d+", version_string) or version_string in ["current"]:
```

Comment:
I may be missing something, but can we use`version_string == "current"` here instead of the list lookup?)

Reply:
Yeah, I thought we also support stable (as an alias for current) so I used list first, but you are right - it's not needed.